lib/ui/src/modules/ui/libs/hierarchy.js

```diff
@@ -46,24 +46,47 @@ function fillHierarchy(namespaces, hierarchy, story) {
   fillHierarchy(namespaces.slice(1), childHierarchy, story);
 }
 
-export function createHierarchy(stories) {
-  const hierarchyRoot = {
+export function createHierarchyRoot(name = '') {
```

Comment:
Looks like you export this function only for tests, IMO it's not a good thing.

Reply:
It's not the main js file, so I think it's Ok to have test-only exports for more granular unit tests

Reply:
👌